### DISCUSSION TITLE
How to create morphology input from raw histology images

### DISCUSSION THREAD

**Discussion by PALMS-gui** [wmglab-duke/ascent#18] [Q&A] [answered]
Could you share some hints how to transform .tiff histology scans into framework inputs (binary maps, scale bar, etc.)
Is there open-source software that can be easily used for that? 

![afbeelding](https://user-images.githubusercontent.com/72300527/152958841-abf76e98-fd3c-4d46-b6c7-5ac3a07711fa.png)

**Comment by thedannymarsh** [comment, marked as answer]
Hi,

We use a few different softwares; people in our lab have used Photoshop, Nikon Elements, and ImageJ (the last one being open source).

Here are some more open source tools for segmentation: https://foobar167.medium.com/open-source-free-software-for-image-segmentation-and-labeling-4b0332049878

Hope that helps!

**Reply by ericmusselman** [reply]
I'll add that our methods for segmenting histology using NIS Elements are described here (Pelot et al., 2020): https://www.frontiersin.org/articles/10.3389/fnins.2020.601479/full
